src/canvas/graphic.ts

```diff
@@ -484,10 +482,10 @@ function bindPathAndTextCommonStyle(
         }
         ctx.globalAlpha = style.opacity == null ? 1 : style.opacity;
     }
-    if (el.hasStroke()) {
```

Comment:
Based on https://deepscan.io/dashboard/#view=project&tid=16023&pid=19250&bid=494172&subview=pull-request&prid=1016382

Reply:
Seems I don't see it in the deepscan result. el can't be null here